pkg/rpc/context.go

```diff
@@ -82,7 +82,7 @@ var SourceAddr = func() net.Addr {
 	return nil
 }()
 
-var enableRPCCompression = envutil.EnvOrDefaultBool("COCKROACH_ENABLE_RPC_COMPRESSION", false)
+var enableRPCCompression = envutil.EnvOrDefaultBool("COCKROACH_ENABLE_RPC_COMPRESSION", true)
```

Comment:
I wonder if this should be changed to a cluster setting while you're mucking with this code.